docs/contributing.md

```diff
@@ -4,7 +4,7 @@ title: Contributing
 slug: /contributing
 ---
 
-The Experimenter documentation hub is managed in the [mozilla/experimenter-docs](https://github.com/mozilla/experimenter-docs) repository. You will need a GitHub account to contribute.
+The Experimenter documentation hub is managed in the [mozilla/experimenter-docs](https://github.com/mozilla/experimenter-docs) repository. You will need a GitHub account to contribute, and if you are not already in the Project Nimbus GitHub team, you may need to request write access in the #nimbus-project Slack channel.
```

Comment:
Should we link the "Project Nimbus GitHub team" text to https://github.com/orgs/mozilla/teams/project-nimbus/members? It took me a hot minute to find the list of members on the GitHub site.

Reply:
Thanks Peter 😁